barcodes_gs1_label/wizard/options_wizard.py

```diff
@@ -26,18 +25,30 @@ class BarcodesGS1PrintOptionsWizard(models.TransientModel):
     show_price = fields.Boolean(string="Show price", default=True)
     show_price_currency = fields.Boolean(string="Show currency", default=True)
     label_copies = fields.Integer(string="Copies", default=1, required=True)
+
+    def _default_barcode_type(self):
+        return (
+            self.env["barcodes.gs1.label.options.config"]
+            .search([("default", "=", True)])
+            .barcode_type
+        )
```

Comment:
Should we add company_id here? Is the company rule applying here in the background?
Is necessary to check if it returns more than one defaults?
Furthermore, maybe adding this logic in the compute?